governance.md

```diff
@@ -0,0 +1,137 @@
+- Release a new npm version of the project on a regular (weekly) basis.
+
+To become an Owner one must fulfill at least the following items and commit to being a part of the community for the long-term.
+
+- Work in a helpful and collaborative way with the MDN community.
+- Have given good feedback on others’ submissions and displayed an overall understanding of the code quality standards for the project.
+- Ability to drive the project forward, manage requirements from users, and taking on responsibility for the overall health of the project.
+
+An individual is invited to become an Owner by existing Owners. A nomination will result in discussion and then a decision by the Owner group.
+
+#### List of Owners
+- Florian Scholz (@Elchi3), Mozilla, BCD project lead
+- Will Bamberg (@wbamberg), Mozilla
+- Chris David Mills (@chrisdavidmills), Mozilla
+- Kadir Topal (@atopal), Mozilla
+
+The owners are also listed as top-level owners in GitHub’s code owner file.
+
+## Project Meetings
+There are no recurrent project meetings, they are scheduled when required, at a time that works for the owners, and using tools that enable participation by the community. The meeting is run by a designated moderator approved by the owners group.
+Items are added to the agenda which are considered contentious or are modifications of governance, contribution policy, owner membership, or release process.
+
+The intention of the agenda is not to approve or review all patches. That should happen continuously on GitHub and be handled by the larger group of Peers. The exception to this is when defining how the schema should look like (or when proposing an update), there is not an initial agreement in the GitHub discussion, and action needs to be taken.
+
+Any community member or Peer can ask that something be added to the next meeting’s agenda by logging a GitHub Issue. Peers can add the item to the agenda by adding the “meeting agenda” tag to the issue and Contributors can ask Peers to add the label for them.
+
+Prior to each project meeting, the moderator will share the agenda with the owners. Owners can add any items they like to the agenda at the beginning of each meeting. The moderator and the owners cannot veto or remove items.
+
+The Owners may invite persons or representatives from certain projects to participate in a non-voting capacity.
+
+The moderator is responsible for summarizing the discussion of each agenda item and adding it to the repository's wiki after the meeting.
+
+### Consensus Seeking Process
+Owners follow a [Consensus-seeking decision-making](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making) model.
+
+When an agenda item has appeared to reach a consensus, the moderator will ask “Does anyone object?” as a final call for dissent from the consensus.
```

Comment:
I'm having a hard time with this bit — this sentence implies that any dissent will stop a motion going forward, and require discussions to resume. But then the following paragraph says that if a consensus (which is basically a majority in this context?) cannot be reached, a vote can be called for to close the issue. Does this mean close it without action, or vote what solution to the issue should be moved forward with? In which case, this is a call for a majority anyway?

Surely we should keep discussing the issue until all points of view have been heard, once that is done put the options for a way forward on the table, and then vote for which option is best in the case of a disagreement? Is this what the text is supposed to be saying anyway? If it is, it is a bit of a confusing way to say it.